project/frontend/src/components/ResultsMap.js

```diff
@@ -8,34 +8,47 @@ const mapStyle = {
 };
 
 function MapContainer(props) {
-  const [activeMarker, changeMarker] = useState({});
-  const [showInfoWindow, changeInfoVisibility] = useState(false);
-
-  const coords = { lat: 40.837, lng: -73.865 };
+  const [activeMarker, setActiveMarker] = useState({});
+  const [showInfoWindow, setShowInfoWindow] = useState(false);
+  // TODO: add a conditional to check if the list is empty
```

Comment:
Sorry to be picky but I think we should add a basic check in this PR (rather than doing it in a separate PR) just so that nothing breaks-- just something simple like 
`if (!restaurants) { 
    alert('No restaurants found.'); 
    return; 
}` 
and we can come back in a separate PR to actually deal with what happens in this case :)

Reply:
yeah im going to do it in this PR i just haven't done it yet because I was working on another PR which is why i added the TODO